README.md

```diff
@@ -1,26 +1,48 @@
+| `limesurvey_mysql_database_name`                   |      `limesurvey`       | The database name                                              |
+| `limesurvey_mysql_user`                            |      `limesurvey`       | Mysql user                                                     |
+| `limesurvey_smtp_server`                           |   :heavy_check_mark:    | SMTP server url                                                |
+| `limesurvey_smtp_port`                             |   :heavy_check_mark:    | SMTP Port                                                      |
+| `limesurvey_smtp_user`                             |   :heavy_check_mark:    | SMTP User                                                      |
+| `limesurvey_smtp_password`                         |            v            | SMTP password                                                  |
+| `limesurvey_smtp_ssl`                              |   :heavy_check_mark:    | SMTP encryption either `ssl`, `tls` or ` `                     |
+| `limesurvey_smtp_debug`                            |           `0`           | SMTP debug option `0` or `1`                                   |
+| `limesurvey_admin_user`                            |         `admin`         | Admin nickname default                                         |
+| `limesurvey_admin_password`                        |   :heavy_check_mark:    | Admin password                                                 |
+| `limesurvey_admin_full_name`                       |       `Your Name`       | Full name of the administrator                                 |
+| `limesurvey_admin_email`                           | `your.name@example.com` | Admin email adress                                             |
+| `limesurvey_mail_admin_name`                       |       `Your Name`       | Sender name for outgoing emails                                |
+| `limesurvey_mail_admin_adress`                     | `your.name@example.com` | Sender adress of outgoing emails                               |
+| `limesurvey_mail_bounce_adress`                    | `your.name@example.com` | Email Adress for bounces                                       |
+| `limesurvey_ldap_plugin_server_url`                |   :heavy_check_mark:    | Url to your ldap server                                                 |
+| `limesurvey_ldap_plugin_server_port`               |          `389`          | Ldap port                                                 |
+| `limesurvey_ldap_plugin_ldap_version`              |   :heavy_check_mark:    | Ldap version either `2` or `3`'                                                 |
+| `limesurvey_ldap_plugin_opt_referrals`             |           `0`           | Select true if referrals must be followed (use false for ActiveDirectory)                                                |
```

Comment:
`true`? Reading the default, I think it should actually be a number?
Please change this to actually work with `True` and `False`.

Reply:
done